### ISSUE TITLE
[FR] - Support for Channels DVR

### ISSUE THREAD

**Issue by daphatty** [vexorian/dizquetv#239]
As the title says, DizqueTV would be great as an addon for Channels DVR. Same premise as Plex, just without Plex. 

Some folks are using DizqueTV in a round about fashion to import M3U channels into Channels DVR, however it requires plex which seems unnecessary.

https://community.getchannels.com/t/how-to-create-your-own-24-7-channels-with-your-own-media/24782

**Comment by vexorian** [comment]
It requires plex for the media library, it's pretty necessary. Eventually there will be ways to play local files without plex, but without the metadata they will not look pretty in the guide.

**Comment by muhlman** [comment]
Yeah, while the Plex requirement isn't ideal for channels it would require pretty significant changes to ditch it. As it stands now it works fine with channels as long as you have Plex, you just need to ensure you set the guide info in channels to custom for the dizque source and import the xmltv in addition to the m3u and you will get all the program specific info instead of just the channel name placeholders.
